components/Layout.tsx

```diff
@@ -5,7 +5,7 @@ import Navbar from "./globals/Navbar";
 import Footer from "./globals/Footer";
 import { NextSeo } from "next-seo";
 
-export const siteTitle = "APIClarity";
+export const siteTitle = "OpenClarity";
 
```

Comment:
This changes the title of the page in the browser bar from API Clarity to OpenClarity:

<img width="480" alt="image" src="https://user-images.githubusercontent.com/1455208/231300741-9d05e393-db2e-4ffa-bf23-ddc6807d2131.png">

Reply:
Current (APIClarity) on the left, new (OpenClarity) on the right.

Reply:
Replaced with new 'eye' favicon:

<img width="483" alt="image" src="https://user-images.githubusercontent.com/1455208/231304784-7107aee5-8cc8-4034-a235-7e8752103d26.png">

<img width="488" alt="image" src="https://user-images.githubusercontent.com/1455208/231304792-6102266b-0725-464b-8aac-10ea23f11782.png">